tests/app/main/test_frameworks.py

```diff
@@ -1613,55 +1614,81 @@ def teardown_method(self, method):
         self.data_api_client.find_draft_services_iter.return_value = [
             {'serviceName': 'A service', 'status': 'submitted', 'lotSlug': 'foo'}
         ]
-        self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(status='complete')
-        self.data_api_client.get_supplier.return_value = SupplierStub().single_result_response()
+        self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(
+            status='complete',
+            application_company_details_confirmed=True,
+        )
+        self.data_api_client.get_supplier.return_value = SupplierStub(
+            company_details_confirmed=True).single_result_response()
 
         self.login()
         res = self.client.get("/suppliers/frameworks/g-cloud-8")
         assert res.status_code == 200
-        assert self.expected in str(res.data)
+
+        document = html.fromstring(res.get_data(as_text=True))
+
+        alert_banner = document.xpath('//div[@class="dm-alert dm-alert--success"]')
+        assert len(alert_banner) == 1
+        assert alert_banner[0].xpath(
+            "//h2[contains(normalize-space(string()), $t)]",
+            t="Your application is complete, and will be automatically submitted.",
+        )
+        assert alert_banner[0].xpath(
+            "//div[contains(normalize-space(string()), $t)]",
+            t="You can change it at any time before the deadline."
+        )
 
         # Check GA custom dimension values
-        doc = html.fromstring(res.get_data(as_text=True))
-        assert len(doc.xpath("//meta[@data-id='29' and @data-value='application_confirmed']")) == 1
+        assert len(document.xpath("//meta[@data-id='29' and @data-value='application_confirmed']")) == 1
+
+    def test_success_banner_with_unsubmitted_drafts_shows_different_message(self, _):
+        self.data_api_client.find_draft_services_iter.return_value = [
+            {'serviceName': 'A service', 'status': 'submitted', 'lotSlug': 'foo'},
+            {'serviceName': 'A service', 'status': 'not-submitted', 'lotSlug': 'foo'}
+        ]
+        self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(
+            status='complete',
+            application_company_details_confirmed=True,
+        )
+        self.data_api_client.get_supplier.return_value = SupplierStub(
+            company_details_confirmed=True).single_result_response()
+
+        self.login()
+        res = self.client.get("/suppliers/frameworks/g-cloud-8")
+        assert res.status_code == 200
+
+        document = html.fromstring(res.get_data(as_text=True))
+
+        alert_banner = document.xpath('//div[@class="dm-alert dm-alert--success"]')
+        assert len(alert_banner) == 1
+        assert alert_banner[0].xpath(
+            "//h2[contains(normalize-space(string()), $t)]",
+            t="Your application is complete, and will be automatically submitted.",
```

Comment:
Suggest amend: Your application is complete and will be submitted automatically.